internal/stationxml/v1.1/email.go

```diff
@@ -0,0 +1,23 @@
+package stationxml
+
+import (
+	"fmt"
+	"regexp"
+)
+
+type Email struct {
+	Address string `xml:",chardata"`
+}
+
+func (e Email) IsValid() error {
+
+	if !(len(e.Address) > 0) {
+		return fmt.Errorf("empty email")
+	}
+
+	if !(regexp.MustCompile(`^[\w\.\-_]+@[\w\.\-_]+$`).MatchString(e.Address)) {
```

Comment:
the `regexp` could be a `const`

Reply:
yes, although that just moves it elsewhere in the code. Again, this code will be removed as I will avoid doing validation prior to encoding (will rely on validators after building).